proto/decentraland/sdk/components/realm_info.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package decentraland.sdk.components;
+
+import "decentraland/sdk/components/common/id.proto";
+option (common.ecs_component_id) = 1106;
+
+message PBRealmInfo {
```

Comment:
what about https://github.com/decentraland/protocol/blob/main/proto/decentraland/kernel/apis/runtime.proto#L64?

Reply:
but this way i have to call this engineApi every time to check if the realm has changed

Reply:
true, go ahead with adding the component, not sure if we should share the body for consistency, or just keep them apart